src/lib/integration_test_local_engine/node_constants.ml

```diff
@@ -0,0 +1,98 @@
+let default_seed_envs =
```

Comment:
You should be able to define these in a more module way. For instance, the ports are all copy-pasted between these, and it's ok to specify some extra env vars that not all nodes need. You could make a `base_mina_node_env` which contains all the ports, the libp2p password, and the privkey password. Then you can have the specialized versions for specific nodes, eg `snark_coord_env`, which extends this base env with the new keys it adds (`let snark_coord_env ... = base_mina_node_env @ [...]`).